src/PactNet.Native/Interop/NativeInterop.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace PactNet.Native.Interop
+{
+    /// <summary>
+    /// Interop definitions to the Pact FFI library
+    /// </summary>
+    internal static class NativeInterop
```

Comment:
Remember when I told you we need to split the interop into a separate file XD good one!